compiler/parser.nim

```diff
@@ -764,14 +777,17 @@ proc commandExpr(p: var Parser; r: PNode; mode: PrimaryMode): PNode =
   result.add commandParam(p, isFirstParam, mode)
 
 proc primarySuffix(p: var Parser, r: PNode,
-                   baseIndent: int, mode: PrimaryMode): PNode =
+                   baseIndent: int, mode: PrimaryMode, 
+                   baseTokType: TokType): PNode =
   #| primarySuffix = '(' (exprColonEqExpr comma?)* ')'
   #|       | '.' optInd symbol generalizedLit?
   #|       | '[' optInd exprColonEqExprList optPar ']'
   #|       | '{' optInd exprColonEqExprList optPar '}'
   #|       | &( '`'|IDENT|literal|'cast'|'addr'|'type') expr # command syntax
   result = r
-
+  if baseTokType == tkStrNumLit:
+    result = dotExpr(p, result)
+    result = parseGStrLit(p, result)
```

Comment:
Seems hacky and has no corresponding grammar update. I'll see if it can be done differently.